content/docs/ops/bosh-releases-in-15-minutes-with-ubuntu.md

```diff
@@ -0,0 +1,143 @@
+```shell
+curl -O http://download.virtualbox.org/virtualbox/5.1.12/virtualbox-5.1_5.1.12-112440~Ubuntu~trusty_amd64.deb
+sudo dpkg -i virtualbox-5.1_5.1.12-112440~Ubuntu~trusty_amd64.deb
+```
+
+### Install Vagrant
+Vagrant provides easy commands to start and manage virtual machine configuration:
+```
+curl -O https://releases.hashicorp.com/vagrant/1.9.1/vagrant_1.9.1_x86_64.deb
+sudo dpkg -i vagrant_1.9.1_x86_64.deb
+```
+
+### Install rbenv
+It is recommended to install Ruby using [rbenv](https://github.com/rbenv/rbenv), which supports multiple parallel ruby installs and keeps all applications from needing to use a single Gem path / specific version.
+```shell
+cd ~
+git clone git://github.com/sstephenson/rbenv.git .rbenv
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
+echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+exec $SHELL
+
+git clone git://github.com/sstephenson/ruby-build.git ~/.rbenv/plugins/ruby-build
+echo 'export PATH="$HOME/.rbenv/plugins/ruby-build/bin:$PATH"' >> ~/.bashrc
+exec $SHELL
+```
+
+### Install Ruby 2.3.1
+Ruby is used for several tools including the BOSH CLI and Vagrant:
+```shell
+rbenv install 2.3.1
+rbenv global 2.3.1
+echo "gem: --no-ri --no-rdoc" > ~/.gemrc
+```
+
+### Install BOSH CLI
+BOSH has a command line client and a server it connects to.  To install the client, which will be the primary interface to BOSH: 
+```shell
+gem install bosh_cli --no-ri --no-rdoc
+```
+
+This client can speak with both BOSH lite and regular BOSH servers.
+
+### Install BOSH Lite
+BOSH Lite provides a lightweight BOSH director running on a Vagrant VirtualBox instance.
+```shell
+mkdir $HOME/workspace
+cd $HOME/workspace
+git clone https://github.com/cloudfoundry/bosh-lite
+cd bosh-lite
+vagrant up
+```
+
+### Connect to BOSH Lite
+Configure the BOSH command line client to connect to the new director running in Vagrant.  The default username is `admin` and password is `admin`:
+```shell
+bosh target 192.168.50.4 lite
+```
+
+### Add Routes
+The VirtualMachine is up and will create containers to service our BOSH requests.  In order for these containers to communicate with the network, a route needs to be added:
+
+```shell
+sudo ./bin/add-route
+```
+
+### Inspect the Director
+The director is up and running and can be inspected to review the configuration.  Currently the director is empty since nothing has been configured or uploaded:
+```shell
+bosh releases
+bosh stemcells
+bosh deployments
+```
+
+### Upload a Release
+[cg-secureproxy-boshrelease](https://github.com/18F/cg-secureproxy-boshrelease) is a BOSH release the cloud.gov team maintains and is a good example.  Clone the release, create a BOSH release and upload it to the director:
+```shell
+cd $HOME/workspace
+git clone https://github.com/18F/cg-secureproxy-boshrelease.git
+bosh create release
+bosh upload release
+```
```

Comment:
Nope, but it's running in production via the Pipeline tasks for [`finalized-bosh-release.*`](https://github.com/18F/cg-pipeline-tasks/)

https://github.com/18F/cg-secureproxy-boshrelease/blob/master/ci/pipeline.yml#L14-L15
[This example is now using the updated bosh-cli written in `golang` v2](https://github.com/18F/cg-pipeline-tasks/blob/master/finalize-bosh-release.sh#L21).